Automattic/wp-calypso
# Component to upload image to media library and related UI

#### Changes proposed in this Pull Request
Implementation to support image upload for a given site during the flow

#### Testing instructions



* Go to the page : `/start/site-content-collection/website-content?siteSlug=<site-slug>`, you should use a valid active site slug!
Eg:  `/start/site-content-collection/website-content?siteSlug=asdas968234344.wordpress.com`

* Make sure the image upload UI is visible
![image](https://user-images.githubusercontent.com/3422709/150948787-57d1fa46-2323-4189-a11f-89ccb8a21c90.png)

* Try uploading an image, a loader should appear when uploading
* After uploading the thumbnail and the caption of the image should be visible

* You should be able to re upload a different image as well 
![image](https://user-images.githubusercontent.com/3422709/150949165-62890f90-8a06-48ed-9fb8-dc28da7aaaf0.png)





Related to #59895
